pkg/api/fed.gateway.solo.io/v1/types/json.gen.go

```diff
@@ -21,93 +20,8 @@ var _ = math.Inf
 
 var (
 	marshaller   = &skv2jsonpb.Marshaler{}
-	unmarshaller = &jsonpb.Unmarshaler{}
+	unmarshaller = &jsonpb.Unmarshaler{
+		AllowUnknownFields: true,
+	}
+	strictUnmarshaller = &jsonpb.Unmarshaler{}
 )
-
-// MarshalJSON is a custom marshaler for FederatedGatewaySpec
-func (this *FederatedGatewaySpec) MarshalJSON() ([]byte, error) {
```

Comment:
ahh.. we may need to upgrade to a more recent skv2 that fixes this

Reply:
this should be fixed in [v0.23.6](https://github.com/solo-io/skv2/releases/tag/v0.23.6)
so that these functions aren't deleted

Reply:
That release of skv2 is on go1.18, and this branch is still on 1.16